app/components/footer_component.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= tag.footer(role: "contentinfo", **html_attributes) do %>
+  <%= tag.div(**container_html_attributes) do %>
+    <% if navigation.present? %>
+      <div class="<%= brand %>-footer__navigation">
+        <%= navigation %>
+      </div>
+
+      <%= tag.hr(class: "#{brand}-footer__section-break") %>
+    <% end %>
+
+    <%= tag.div(class: meta_classes, **meta_html_attributes) do %>
+      <% if meta.present? %>
+        <%= meta %>
+      <% else %>
+        <div class="<%= brand %>-footer__meta-item <%= brand %>-footer__meta-item--grow">
+          <% if meta_pre_content.present? %>
```

Comment:
What meta data are in these things? `meta_pre_content` and `meta`

Reply:
This is mostly a hard copy from the https://github.com/x-govuk/govuk-components/blob/main/app/components/govuk_component/footer_component.html.erb.

I've only just slotted in that additional `meta_pre_content` section.

Reply:
Ah OK, thanks 👍🏻